.travis.yml

```diff
@@ -6,10 +6,10 @@ python:
 - "3.7"
 services: docker
 before_install:
-- sudo wget -O- http://neuro.debian.net/lists/xenial.us-ca.full | sudo tee /etc/apt/sources.list.d/neurodebian.sources.list
-- sudo apt-key adv --recv-keys --keyserver hkp://pool.sks-keyservers.net:80 0xA5D32F012649A5A9 
 - sudo apt-get update
-- sudo apt-get install -y singularity-container
+- sudo apt-get install neurodebian-popularity-contest
```

Comment:
This seems to have slipped in; shouldn't be needed.

Reply:
This is required as otherwise i was getting dependency error. `unment dependency : neurodebian-popularity-contest`

Reply:
that might be solved with `apt-get -f install` after doing `apt install`